client/proposer-metrics/src/lib.rs

```diff
@@ -47,6 +48,8 @@ impl MetricsLink {
 pub struct Metrics {
 	pub block_constructed: Histogram,
 	pub number_of_transactions: Gauge<U64>,
+	pub hit_block_size_limit: CounterVec,
+	pub hit_block_weight_limit: CounterVec,
```

Comment:
I believe `oversized_block_count` and `overweight_block_count` work better in this case.

Reply:
Or even better: oversized_block_error_count and overweight_block_error_count